src/ontology/cl-edit.owl

```diff
@@ -30663,6 +30664,17 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29904965"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_4032001 "reelin GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4032001 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013879)))
 
+# Class: obo:CL_4033014 (peg cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "http://purl.obolibrary.org/obo/BTO_0005588"^^xsd:string) obo:IAO_0000115 obo:CL_4033014 "A non-ciliated secretory epithelial cell that is part of some oviduct.")
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033014 "2023-02-02T15:07:12Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4033014 "intercalary cell")
+AnnotationAssertion(rdfs:label obo:CL_4033014 "peg cell")
+SubClassOf(obo:CL_4033014 obo:CL_0000151)
```

Comment:
Consider removing, unless you add evidence that this cell functions as a 'secretory cell' as it is defined in CL.

Reply:
It is not a secretory cell, so I have removed it. The literature that was considering them secretory is outdated